spec/rspec/rails/matchers/active_job_spec.rb

```diff
@@ -98,6 +98,31 @@ def self.name; "LoggingJob"; end
       expect { }.not_to have_enqueued_job
     end
 
+    context "when job is retried" do
+      include ActiveJob::TestHelper
+
+      let(:retried_job) do
+        Class.new(ActiveJob::Base) do
+          retry_on StandardError, queue: :retry
+
+          def self.name; "RetriedJob"; end
+          def perform; raise StandardError; end
+        end
+      end
+
+      before { stub_const("RetriedJob", retried_job) }
+
+      it "passes with reenqueued job" do
+        retried_job.perform_later
+
+        time = Time.current.change(usec: 0)
+        travel_to time do
+          expect { perform_enqueued_jobs(queue: :default) }
+            .to have_enqueued_job(retried_job).on_queue(:retry).at(time + 3)
```

Comment:
I don't get it. Does this mean 3 seconds later? Where does this 3 come from?

Reply:
default, I'll change it to be explicit `retry_on …,wait: 3` along with dropping unrelated stuff.